lib/site-options.php

```diff
@@ -59,6 +59,49 @@ function igv_register_theme_options_metabox() {
     ),
   ) );
 
+  // Modules
+  $modules_group = $front_page_options->add_field( array(
```

Comment:
I think this should be a post type. It seems enough to be classes as a type of post esp in the future with another iteration of this website they might even get their own pages each.